data_model/src/parameter.rs

```diff
@@ -344,45 +363,86 @@ impl SumeragiParameters {
     }
 }
 
+mod defaults {
+    pub mod sumeragi {
+        pub fn default_block_time_ms() -> u64 {
+            2_000
+        }
+        pub fn default_commit_time_ms() -> u64 {
+            4_000
+        }
+        pub fn default_max_clock_drift_ms() -> u64 {
+            1_000
+        }
+    }
+
+    pub mod block {
+        use core::num::NonZeroU64;
+
+        use nonzero_ext::nonzero;
+
+        /// Default value for [`Parameters::MaxTransactionsInBlock`]
+        pub fn default_transactions_in_block() -> NonZeroU64 {
+            nonzero!(2_u64.pow(9))
+        }
+    }
+
+    pub mod transaction {
+        use core::num::NonZeroU64;
+
+        use nonzero_ext::nonzero;
+
+        pub fn default_instruction_number() -> NonZeroU64 {
+            nonzero!(2_u64.pow(12))
+        }
+        pub fn default_smart_contract_size() -> NonZeroU64 {
+            nonzero!(4 * 2_u64.pow(20))
+        }
+    }
+
+    pub mod smart_contract {
+        use core::num::NonZeroU64;
+
+        use nonzero_ext::nonzero;
+
+        pub fn default_fuel() -> NonZeroU64 {
```

Comment:
```suggestion
        pub const fn default_fuel() -> NonZeroU64 {
```

can't it be `const fn`?